samples/shared-heap/wasm-apps/test1.c

```diff
@@ -14,12 +15,41 @@ shared_free(void *ptr);
 void *
 my_shared_malloc(uint32_t size, uint32_t index)
 {
-    char *buf = shared_malloc(size);
+    char *buf1 = NULL, *buf2 = NULL, *buf;
```

Comment:
redefinition of 'buf'

Reply:
here it defines buf1, buf2 and buf, should not be an issue.

Reply:
Ok, it's my mistake